### PR TITLE
explain the context about the new transaction queues and failed trx billing

### PR DESCRIPTION
**Pull request by Adam-Ultra**
explain more about why we introduce the new transaction queues and failed transaction billing, why subjective billing is not enough in the context of ultra. The background we introduced the features, and how they work with each other.

### CODE REVIEW

**Review by janos-ultra — COMMENTED** [review]
Overall looks good, I left some small comments

File: docs/learn/Ultra Blockchain/Transaction-queues-billing-and-rate-limiting.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+3. The transaction reaches the block producer nodeos, gets executed, and is included in a block.
+
+However, many transactions sent to API nodeos and block producer nodeos pass signature validation but fail internal execution checks, like assert failure. This overflow of failed transactions puts strain on block producers and API nodeos, affecting the CPU time usage for regular transactions.
+
+# Subjective Billing
+
+To address this, subjective billing was introduced. It aims to bill failed transactions and reject any accounts that go subjectively negative. Successful transactions replace their temporary subjective billing with objective billing upon blockchain recording. While this approach significantly reduced CPU waste, it can result in inconsistent billing across nodeos. Consequently, transactions might be lost at any point, potentially causing an account to encounter recurring resource errors and subsequent subjective billings. [For a deeper dive, refer to this article.](https://eosnetwork.com/blog/api-plus-an-introduction-to-subjective-billing-and-lost-transactions/ "An Introduction to Subjective Billing and Lost Transactions").
+
```
[the remark this answers is not in the document]
Comment (Adam-Ultra) [reply docs/learn/Ultra Blockchain/Transaction-queues-billing-and-rate-limiting.md:23]: updated

File: docs/learn/Ultra Blockchain/Transaction-queues-billing-and-rate-limiting.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Two Transaction Queues
+
+To balance free network usage for end users and ensure staked users access their resources, two transaction queues have been introduced:
+
+1. **Staked User Queue**: For users with staking (primarily developers). This queue can use up to a set CPU time percentage per block (e.g., 80%).
+
+2. **Free User Queue**: For users without staking. This queue can use the remaining CPU time (e.g., 20%). If one queue is vacant, the other can use more CPU time.
+
+To counter spamming from non-staking users, there's a rate limit. Exceeding a set CPU usage threshold results in users getting rate-limited. Higher transaction demands necessitate more staking. [See more details here](./rate-limiting.html)
+
```
[the remark this answers is not in the document]
Comment (Adam-Ultra) [reply docs/learn/Ultra Blockchain/Transaction-queues-billing-and-rate-limiting.md:37]: updated

**Review by janos-ultra — APPROVED** [review]
LGTM. I haven't moved the ticket to "Done" because the BPs or Rami might have some comments on this documentation.

**Pull request merged**